decidim-core/config/locales/en.yml

```diff
@@ -1262,6 +1262,14 @@ en:
           invitation_instructions:
             accept_until_format: "%B %d, %Y %I:%M %p"
       time_of_day: "%H:%M"
+  versions:
+    directions:
+      left: Deletions
+      right: Additions
+    dropdown:
+      choose_diff_view_mode: 'Diff view mode:'
```

Comment:
Maybe `Diff` it's too technical? What about `Compare`? /cc @decidim/product

Reply:
Agree with @oriolgual, `compare` better than `diff`

Reply:
```suggestion
      choose_diff_view_mode: 'Compare view mode:'
```